cli/dcoscli/package/main.py

```diff
@@ -763,7 +763,8 @@ def _get_package_manager():
     if cosmos_manager.enabled():
         return cosmos_manager
     else:
-        msg = ("This version of the dcos-cli is unsupported for your DCOS "
-               "cluster. Please use a dcos-cli version < 0.4.0 or upgrade your"
-               " cluster to DCOS version >= 1.6.1")
+        msg = ("This version of the DCOS CLI is not supported for your "
+               "cluster. Please downgrade the CLI to an older version: "
+               "https://docs.mesosphere.com/administration/introcli/updatecli/"
```

Comment:
@jsancio @tamarrow Is this good enough, or does it actually need it's own page: https://docs.mesosphere.com/administration/cli/updatecli/#scrollNav-2.